doc/api/fs.markdown

```diff
@@ -350,6 +350,10 @@ created. On POSIX systems, `path` is considered to exist even if it is a symlink
 to a non-existent file. The exclusive flag may or may not work with network file
 systems.
 
+`flags` can also be a number as documented by open(2); commonly used constants
```

Comment:
Well, there are many (2) notations in this file; do you want to linkify every one?  I suppose that should be another PR anyway.